internal/middleware/logging.go

```diff
@@ -2,65 +2,95 @@ package middleware
 
 import (
 	"context"
-	"time"
+	"encoding/json"
 
 	"github.com/openfga/openfga/pkg/logger"
 	serverErrors "github.com/openfga/openfga/pkg/server/errors"
+	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 )
 
+const (
+	methodKey       = "method"
```

Comment:
`grpc_method` just to be precise. That will also play nicely with the other standards in the `grpc-middleware` ecosystem.

Reply:
Done. Thanks!